app/modules/google/google.redux.js

```diff
@@ -0,0 +1,21 @@
+import { createActions, createReducer } from 'reduxsauce';
+import { Record, List } from 'immutable';
+
+
+export const { Types: GoogleTypes, Creators: GoogleActions } = createActions({
+  getPlaces: ['latitude', 'longitude', 'placeType', 'name'],
+  getPlacesSuccess: ['data'],
+  getPlacesFailure: ['error'],
+}, { prefix: 'GOOGLE_' });
+
+const GoogleRecord = new Record({
+  places: List(),
+});
+
+export const INITIAL_STATE = new GoogleRecord({});
+
+const getPlacesSuccessHandler = (state, { data: { results } }) => state.merge({ places: results });
```

Comment:
skoro pobieramy różne typy places per typ i wszysctkie trzymamy w obiekcie places to może warto zrobić coś takiego? 
`
state.setIn(['places', type], fromJS(results))
`

Wtedy jeszcze parametryczny selektor do wyciągania places po typie i mamy większość komponentów ogarniętych jednym reducerem